dotcom-rendering/src/components/GetMatchNav.importable.tsx

```diff
@@ -38,20 +56,16 @@ export const GetMatchNav = ({
 	tags,
 	webPublicationDateDeprecated,
 }: Props) => {
-	const options: SWRConfiguration = { errorRetryCount: 1 };
-	// If this blog is live then poll for new stats
-	if (format.design === ArticleDesign.LiveBlog) {
-		options.refreshInterval = 16_000;
-	}
-	const { data, error, loading } = useApi<{
-		homeTeam: TeamType;
-		awayTeam: TeamType;
-		comments?: string;
-		minByMinUrl?: string;
-		venue?: string;
-	}>(matchUrl, options);
+	const options = {
+		errorRetryCount: 1,
+		fallbackData,
+		refreshInterval:
+			// If this blog is live then poll for new stats
+			format.design === ArticleDesign.LiveBlog ? 16_000 : undefined,
```

Comment:
Good idea!

Reply:
This was already present, but now folded inside the initial config rather than relying on mutation. It enables the use of the `satisfies` keyword.